packages/discord.js/src/structures/Emoji.js

```diff
@@ -46,7 +46,7 @@ class Emoji extends Base {
    * @returns {?string}
    */
   imageURL(options) {
-    return this.id && this.client.rest.cdn.emoji(this.id, options);
+    return this.id && this.client.rest.cdn.emoji(this.id, this.animated ? { extension: 'gif', ...options } : options);
```

Comment:
Actually, while this works, it might be worth adding support for the `forceStatic` option like the other dynamic image URL methods

You'd have to change the parameter type to `ImageURLOptions` and handle the `forceStatic` option here as well

Reply:
what about adding an `animated` option for the `emoji` function in /rest?

Reply:
That's fine by me